addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/PentairPacket.java

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pentair;
```

Comment:
you might want to move such internal classes to an internal package such as org.openhab.binding.pentair.internal